apiv2/src/plan-marketing/infra/api/Campagne.controller.ts

```diff
@@ -12,12 +12,17 @@ export class CampagneController {
         @Inject(CampagneGateway) private readonly campagneGateway: CampagneGateway,
         private readonly campagneService: CampagneService,
     ) {}
-
+    
     @Post()
     async create(@Body() dto: CreateCampagneDto): Promise<CampagneModel> {
         return await this.campagneService.creerCampagne(dto);
     }
-
+    
+    @Get("all")
```

Comment:
```suggestion
    @Get()
```
un GET sur `/compagne`, sous entend que l'on récupère tout, on met un suffix pour les actions spécifiques en POST.
- liste des campagnes: `GET /compagne`
- récupération campagne unique : `GET /compagne/:id`